rfcs/0013-data-plane-fanout.md

```diff
@@ -0,0 +1,86 @@
+The retransmit mechanism also supports a second, `grow`,  mode of operation that squares the number of neighborhoods allowed per layer which dramatically reduces the number of layers needed to support a large network but can also have a negative impact on the network pressure each node in the lower layers has to deal with.  
+
+Below is an example of a two layer network. Note - this example doesn't describe the same `fanout/2` limit for lower layer neighborhoods.
+               
+                                              +--------------+
+                                              |              |
+                                 +------------+    Leader    +------------+
+                                 |            |              |            |
+                                 |            +--------------+            |
+                                 v                                        v
+                        +--------+--------+                      +--------+--------+
+                        |                 +--------------------->+                 |
+      +-----------------+   Validator 1   |                      |   Validator 2   +-------------+
+      |                 |                 +<---------------------+                 |             |
+      |                 +------+-+-+------+                      +---+-+-+---------+             |
+      |                        | | |                                 | | |                       |
+      |                        | | |                                 | | |                       |
+      |                +---------------------------------------------+ | |                       |
+      |                |       | | |                                   | |                       |
+      |                |       | | |            +----------------------+ |                       |
+      |                |       | | |            |                        |                       |
+      |                |       | | +--------------------------------------------+                |
+      |                |       | |              |                        |      |                |
+      |                |       | +----------------------+                |      |                |
+      v                v       v                v       v                v      v                v
+    +-+----------------+-+   +-+----------------+-+   +-+----------------+-+  +-+----------------+-+
+    |                    |   |                    |   |                    |  |                    |
+    |   Neighborhood 1   |   |   Neighborhood 2   |   |   Neighborhood 3   |  |   Neighborhood 4   |
+    |                    |   |                    |   |                    |  |                    |
+    +--------------------+   +--------------------+   +--------------------+  +--------------------+
+              
+#### A Stake weighted selection mechanism
```

Comment:
It could be more generic and just call it a "weighted" selection mechanism.  The "Stake" part is consensus specific.

Reply:
yes, I think we wanted to make this more intelligent in the future anyway. I'll update it.